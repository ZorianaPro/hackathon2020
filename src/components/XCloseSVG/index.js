import React from 'react';

const XCloseSVG = (props) => {
  return (
    <svg className="XCloseSVG"
      {...props}>
      <use xlinkHref="#XCloseSymbol" />
    </svg>
  );
};

export default XCloseSVG;
