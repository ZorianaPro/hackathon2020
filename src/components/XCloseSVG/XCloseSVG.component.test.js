import React from 'react';
import { shallow } from 'enzyme';
import XCloseSVG from './';

describe('XCloseSVG', () => {
  it('renders without crashing', () => {
    expect(shallow.bind(shallow, <XCloseSVG />)).not.toThrow();
  });

  it('renders as expected', () => {
    expect(shallow(<XCloseSVG />)).toMatchSnapshot();
  });
});
