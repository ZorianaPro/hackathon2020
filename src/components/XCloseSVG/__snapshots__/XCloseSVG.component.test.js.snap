// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`XCloseSVG renders as expected 1`] = `
<svg
  className="XCloseSVG"
>
  <use
    xlinkHref="#XCloseSymbol"
  />
</svg>
`;
